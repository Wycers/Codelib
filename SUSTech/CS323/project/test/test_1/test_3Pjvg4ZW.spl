int test_11711335_2(float a, char b){
    int result = 0XCcLd143;
    return i * a;
}
