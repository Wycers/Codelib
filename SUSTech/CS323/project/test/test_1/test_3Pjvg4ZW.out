Error type A at Line 2: unknown lexeme 0XCcLd143
